velox/common/base/SimdUtil.h

```diff
@@ -496,7 +496,8 @@ xsimd::batch<T, A> reinterpretBatch(xsimd::batch<U, A>, const A& = {});
 // equal. May address up to SIMD width -1 past end of either 'x' or 'y'.
 template <typename A = xsimd::default_arch>
 inline bool memEqualUnsafe(const void* x, const void* y, int32_t size);
-
+FOLLY_ALWAYS_INLINE size_t
```

Comment:
Add empty line and document this function. Make sure to spell out parameter names. Do not use single-letter names. Remove 'simd' prefix from the function name. This function is in velox::simd namespace, hence, "simd" is implied.